frontend/src/pages/notebookController/screens/server/SpawnerPage.tsx

```diff
@@ -79,8 +78,7 @@ const SpawnerPage: React.FC = React.memo(() => {
     image: undefined,
     tag: undefined,
   });
-  const [sizeDropdownOpen, setSizeDropdownOpen] = React.useState<boolean>(false);
-  const [selectedSize, setSelectedSize] = React.useState<string>('');
+  const [selectedSize, setSelectedSize] = React.useState<string>('Default');
```

Comment:
Maybe we just go with something like:

```
const [selectedSize, setSelectedSize] = usePreferredNotebookSize()
```

Then the hook would do the following:
- Look for their preferred notebook size (in Dashboard Config today, in Notebook "tomorrow" - #416)
- If they don't have one -- use the first item in the Dashboard Config `notebookSizes`
- For #303 -- We can wrap the setter function so when they call it with their new selection we store it in localStorage and we add it to our logic for the first bullet (for new pages)

It's all contained, no onus on the SpawnerPage to worry about such things.